pages/osx/mkfile.md

```diff
@@ -0,0 +1,15 @@
+# mkfile
+
+> Create one or more empty files of any size.
+
+- Create an empty file of 15 kilobytes:
+
+`mkfile -n {{15k}} {{file_name}}`
+
+- Create a file of a given size and unit (bytes, kB, MB, GB):
+
+`mkfile -n {{size}}{{b|k|m|g}} {{file_name}}`
+
+- Create two files of 4 megabytes:
```

Comment:
```suggestion
- Create 4 megabyte files:
```

Perhaps this will read better?

Reply:
I'm not sure.. I'm not a native English speaker, but I would interpret it as "create 4 files of 1 megabytes each"
If it's only me, I'll be happy to apply your suggestion

Reply:
`Create 2 files of 4 megabytes each` would work too :-)